src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -73,6 +108,8 @@
       <PreviouslySourceBuiltNuGetSourceName>previously-source-built</PreviouslySourceBuiltNuGetSourceName>
       <ReferencePackagesNuGetSourceName>reference-packages</ReferencePackagesNuGetSourceName>
       <SourceBuildSources>$(SourceBuildSources);$(PrebuiltNuGetSourceName);$(PreviouslySourceBuiltNuGetSourceName);$(ReferencePackagesNuGetSourceName)</SourceBuildSources>
+      <BuildCommand>$(BuildCommand) /p:RestoreConfigFile=@(NuGetConfigFiles)</BuildCommand>
```

Comment:
Doesn't this need to happen for non-sources only flows as well?  Aren't the local current package feeds needed to flow cross repo dependencies?

Reply:
Yes correct.

Reply:
Will move it - thanks.

Reply:
Fixed with https://github.com/dotnet/installer/pull/18478/commits/05164e106ecff2a763c3d52dd13c328dbb35e485